src/exec-ram.c

```diff
@@ -137,10 +137,12 @@ static ram_addr_t find_ram_offset(ram_addr_t size) {
     }
 
     if (best_size == UINT_MAX) {
+        fprintf(stderr, "libcpu: ram size too large\n");
         abort();
     }
 
     if (best_offset == RAM_ADDR_MAX) {
+        fprintf(stderr, "libcpu: ram size too large\n");
```

Comment:
More precisely this error means that there is no free slot available in the list of ram blocks.